test/project_types/script/forms/ask_app_test.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require "project_types/script/test_helper"
+
+describe Script::Forms::AskApp do
+  include TestHelpers::FakeFS
```

Comment:
I'm not sure why `FakeFS` is needed here?  Is it because we pass in `Dir.mktmpdir` on line 8?  It feels like `context` can just be a `Mock`.  Anyway, don't worry about making changes to this comment.  But this question crossed my mind.

Reply:
I think this was actually a holdover from the old ensure env test I extracted these from.

I don't think they are needed in the forms since the forms are not doing any FS operations, just returning a form object with a value. I've removed the Dir call and this include from all the forms.